### ISSUE TITLE
"Desm" in the title attribute should be "DESM"

### ISSUE THREAD

**Issue by jeannekitchens** [t3-innovation-network/desm#366]
From @siuc-nate

@jbaird123 this should be assigned to Alex to correct. 

![image](https://user-images.githubusercontent.com/20917206/206200677-a2bcf8db-dc9a-446b-a800-74107fd4bf02.png)

**Comment by jbaird123** [comment]
Resolved
